scripts/complete_liquidity_provision.js

```diff
@@ -64,15 +67,24 @@ module.exports = async callback => {
 
     assert(argv.fleetSize % 2 == 0, "Fleet size must be a even number for easy deployment script")
 
-    console.log("1. Check for sufficient funds")
+    console.log("1. Sanity checks")
     if (!(await checkSufficiencyOfBalance(targetToken, masterSafe.address, investmentTargetToken))) {
-      callback(`Error: MasterSafe has insufficient balance for the token ${targetToken.address}.`)
+      return callback(`Error: MasterSafe has insufficient balance for the token ${targetToken.address}.`)
```

Comment:
In the [Truffle script documentation](https://www.trufflesuite.com/docs/truffle/getting-started/writing-external-scripts) it's stated that `callback` is supposed to be called at the end of the script, using as optional input an error message. So the `return` should not be needed here.